workspaces/liferay-customer-workspace/client-extensions/liferay-customer-etc-spring-boot/src/main/java/com/liferay/customer/JiraRestController.java

```diff
@@ -472,8 +480,8 @@ private JSONObject _transformIssueFields(JSONObject issueFieldsJSONObject) {
 				issueFieldsJSONObject.getJSONArray(_FIELD_VERSIONS))
 		).put(
 			"category",
```

Comment:
@jennyhchen should we rename the field to categories now? since it can be more than one?